guides/common/modules/proc_creating-a-webhook-template.adoc

```diff
@@ -20,3 +20,30 @@ For more information, see {ManagingHostsDocURL}Template_Writing_Reference_managi
 . Optional: Enter the description and audit comment.
 . Assign organizations and locations.
 . Click *Submit*.
+
+.Examples
+When creating a webhook template, it is important to follow the format of the target application that the template is made for.
+For example, application can expect a `"text"` field with the webhook message.
+Refer to the documentation of your target application to find more about how your webhook template format should look like.
+
+Example 1::
+This webhook template carries a message with id and result of a remote execution job.
```

Comment:
```suggestion
This webhook template contains a message with the ID and result of a remote execution job.
```

Reply:
Hmm, on second thought: the template does not actually contain a message. instead, it's the blueprint for the payload of the API call that Foreman makes. So maybe "the template defines the message that {Project} sends ..."?